### PR TITLE
Create README.md

### PR DESCRIPTION
**Pull request by Jin-prgrmng**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+HTML&CSS
+JQuery
+
+#purpose of the project
+To make them aware me and get job.
+
+#Link to the hosted web page of this folder
+https://jin-prgrmng.github.io/my_portfolio_site
+
+#Authors
```
Comment (oladhari) [inline README.md:14]: just mention yourself here, as it is your personal project

File: README.md [hunk 2]
```diff
@@ -0,0 +1,22 @@
+#Project Title
+Jin's Portfolio
+
+#Used technologies
+HTML&CSS
+JQuery
+
+#purpose of the project
+To make them aware me and get job.
```
Comment (oladhari) [inline README.md:9]: test a new portfolio website

File: README.md (hunk 1 above)
Comment (Jin-prgrmng) [reply README.md:14]: Ok. I'll correct  it.

File: README.md (hunk 2 above)
Comment (Jin-prgrmng) [reply README.md:9]: Not working...

**Pull request merged**